### PR TITLE
July 28 Release Notes

### PR DESCRIPTION
**Pull request by lbadenhop**
Thank you for your contribution to the Amplify-Central repo.

## Describe the changes

Enter a brief description of your changes to communicate to the reviewers what you changed and why.

## Deploy preview link

Please add the deploy preview link to the **specific page** that you've changed.

## Checklist for contributors

Before submitting this PR:

* Verify that all status checks have passed
* For more information on the Markdown linter rules, see [DavidAnson/markdownlint](https://github.com/DavidAnson/markdownlint)
* For first-time contributors, ensure that you have signed the [Axway CLA](https://cla.axway.com/)

### CODE REVIEW

**Comment by docsbuilder** [comment]
### <span aria-hidden="true">✅</span> Deploy preview is ready!

 |  Name | Link |
 |---------------------------------|------------------------|
 |<span aria-hidden="true">🔨</span> Latest commit | 1acacae |
 |<span aria-hidden="true">😎</span> Deploy preview | http://PR-328.amplify-central.opendocs-builder.pcloud.axway.int |

File: content/en/docs/amplify_relnotes/20230728_amplify.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+
+#### Axway API Management Gateway
+
+There are no enhancements or bug fixes in this release.
+
+#### Embedded agent
+
+##### Enhancements for Embedded Agent
+
+* **Embedded AWS Traceability Agent** - A new Embedded Agent traceability service for AWS Gateway is now available.  The traceability service reqires an Access Log ARN (Amazon Resource Name) to access transaction events in CloudWatch.  Use Central CLI (**URL link to NPM**) or the Central WebUI to create the new AWS Gateway environment with an Embedded Agent traceability service.
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20230728_amplify.md:38]: requires

File: content/en/docs/amplify_relnotes/20230728_amplify.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+
+##### Enhancements for Embedded Agent
+
+* **Custom Settings for the Embedded AWS Agent** - The Central WebUI, create/edit Environments page, can be used to set customer settings for either the Discovery or Traceabiliey service.   For example, this includes the Team Ownership or filter by tags optional settings.
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20230728_amplify.md:42]: custom settings
Traceability

File: content/en/docs/amplify_relnotes/20230728_amplify.md [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+
+The following enhancements and bug fixes are available in this release.
+
+#### Feature updates for Product Foundry
+
+There are no new features in this release.
+
+#### Bug fixes for Product Foundry
+
+* **Subscriptions and accessRequest filter does not show all the team** - Subscriptions and AccessRequest list now display in the team filter all the teams present in the system to allow providers to filter correctly the subscription and access request.
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20230728_amplify.md:124]: **Subscriptions and accessRequest filters do no show all teams** - The team filter displayed in both the Subscriptions and the Access Requests screens now includes all teams available in the platform.

File: content/en/docs/amplify_relnotes/20230728_amplify.md [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+* **Plan status update** - an automatic refresh is performed once a plan gets deprecated to show correctly its new status to the user.
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20230728_amplify.md:125]: **Plan status update** - When a plan gets deprecated, the table is automatically refreshed to display the proper plan status.

File: content/en/docs/amplify_relnotes/20230728_amplify.md [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+* **Product visibility screen for private marketplace is missing the consumption preferences** - The missing field are now visible regardless of the marketplace type (public / private).
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20230728_amplify.md:126]: The missing configuration is now visible regardless of the marketplace type (public / private).

File: content/en/docs/amplify_relnotes/20230728_amplify.md [hunk 6]
```diff
@@ -0,0 +1,159 @@
...
+* **Product activation failure after removing product documentation overview** - the remove of product document from the document overview is now cleaning the resource correctly and does not prevent to create a new product release.
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20230728_amplify.md:127]: The removal of a product document from the document overview is no longer preventing the user from creating a new product release.

File: content/en/docs/amplify_relnotes/20230728_amplify.md [hunk 7]
```diff
@@ -0,0 +1,159 @@
...
+* **Impossible to archive a deprecated plan when attached to migrated subscription** - The archival of a plan is now possible even if there are some migrated subscriptions attached to it. The plan archival will force the archive of associated migrated subscriptions. A confirmation popup listing the remaining migrated subscriptions is displayed for the user to confirm the action.
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20230728_amplify.md:128]: **Impossible to archive a deprecated plan when attached to migrated subscriptions** - The archival of a plan is now possible even if there are still associated subscriptions in migrated states. The plan archival will force the archive of associated migrated subscriptions. A confirmation popup listing the remaining migrated subscriptions is displayed for the user to confirm the action.

**Review by georgiana-caras — CHANGES_REQUESTED** [review]

**Review by cbordier — APPROVED** [review]
I approve the changes

**Pull request merged**